### DISCUSSION TITLE
Dedicated Base for 500w Power Supply

### DISCUSSION THREAD

**Discussion by DannyKlenz** [jon-harper/OmniBox#100] [Ideas]
Would you be willing to give the 500w power supply its own part that doesn't require the extensions?

**Comment by jon-harper** [comment]
Not in the short term, but it's something I will consider for after 1.0. The RSP-500 is just so much bigger than other PSUs.

Some users have tossed ideas around for other purposes that a taller base might be used for (storage, etc.). If that comes to fruition, I'd be willing to consider it sooner.

**Comment by jon-harper** [comment]
Another user pointed out that the PSU is only used by Ender 5 Plus users, who can print the entire piece at once. It makes way too much sense not to issue a monolithic piece from that perspective: front and rear base and shim.

I'll get to work on this before next release.

**Reply by DannyKlenz** [reply]
My apologies I think there might be a misunderstanding. What i was requesting is that the power supply section that the switch and plug mount to, be made taller so only the shims are no longer necessary. Everything else is great! I figured since there was already a unified power supply section that making it a little taller would not be much of an issue. I do understand that if there is not much use for it then it doesn't make sense to go through the effort.

**Reply by jon-harper** [reply]
It's my wording that is confusing, I believe. To clarify:

Normally there is a front base, rear base, and matching shim pieces, for a total of four parts. I believe we are both suggesting that there should be no shim, just one piece with a taller base, like this:

![image](https://user-images.githubusercontent.com/11852387/227214068-d8f2cbb2-c452-4439-9aca-b36a963c680c.png)
![image](https://user-images.githubusercontent.com/11852387/227214167-f3e37a84-16ed-41ee-8cdf-4d3e199f2673.png)

No more shim, taller, and one piece. 😃 

You can get the untested file here: [Base - RSP500 - HSI.stl](https://github.com/jon-harper/OmniBox/blob/unstable/Core/Base/Unified/Base%20-%20RSP500%20-%20HSI.stl)

**Comment by TheoBromine2022** [comment]
> Would you be willing to give the 500w power supply its own part that doesn't require the extensions?

https://old.reddit.com/user/Neoliberal_Boogeyman/comments/12ophd5/rsp50024_power_supply_enclosure_for_the_omnibox/

Does this meet your needs? the files are on the omnibox discord.
